### PR TITLE
To Do list: add & remove

### PR DESCRIPTION
**Pull request by denisdiaconu**
implemented the CRUD (create, update, delete) methods

### CODE REVIEW

File: src/dom.js [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+
+export const clearAll = () => {
+  const clearCheckedBtn = document.getElementById('clearChecked');
+  clearCheckedBtn.addEventListener('click', () => {
+    const list = getList();
+    const newList = list.filter((todo) => todo.completed !== true);
+    updateList(newList);
+    window.location.reload();
+  });
+};
```
Comment (ZawadiSumba66) [inline src/dom.js:81]: - [x] When I check a few several tasks or even a single task and click on the clear completed tasks button, the tasks get unchecked. However, the checked tasks should be removed from the task list. Please fix this.

**Review by ZawadiSumba66 — CHANGES_REQUESTED** [review]
Hi @denisdiaconu 👋 , 

Great job so far on your project. 👏 👏 
However, there are some issues that you still need to work on to go to the next step but you are almost there! 💪 💪 

## Required Changes ♻️
- [x] Kindly add an `npm start` in your README at the end of the getting started section to start the server.

_Please check the other comment under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by denisdiaconu** [comment]
Added an npm start in the README.
Fix the remove completed button.

**Review by mricanho — APPROVED** [review]
# Status: ✔️ Project Approved ✔️ 

Hello @denisdiaconu🔥 

You worked hard in all the instructions that the previous Code Reviewers give you👍🏽 

doing good

Now you can merge this branch 🚀 

### 💻 👩🏽‍💻  Happy Coding!  👨🏽‍💻 🖥️

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**